client/src/views/CodeView.vue

```diff
@@ -3,30 +3,23 @@
     <div class="col-12" style="margin-left: 3vh">
       <div class="container">
         <div class="row" style="margin-top: 30px">
-          <div class="col-5" style="padding: 0px; display: flex">
-            <button
-              type="button"
-              class="btn btn-primary btn-sm"
-              @click="generate"
-            >
-              Generate
-            </button>
-            <SqlSchema
-              :showSchemaOption="language == 'sql'"
-              ref="schema"
-              style="margin-left: 1.5vh"
-              @updateSchemaStatus="updateStatus"
-            />
-          </div>
-          <div class="col-5" align="right" style="padding: 0px">
-            <ValidationLevel ref="override_level" />
+          <div class="col-5" style="padding: 0px">
+            <div class="row" style="margin-left:1%; font-size: small">
+              Validation override level
```

Comment:
It's still not intuitive what this means.  Maybe something like "Substrait Validator Errors to Ignore"?

Reply:
made the change. Thanks!